docs/end-to-end-demo.md

```diff
@@ -2,6 +2,11 @@
 In this guide, you will deploy Akri end-to-end, all the way from discovering local video cameras to the footage being streamed on a Web application. You will explore how Akri can dynamically discover devices, deploy brokers pods to perform some action on a device (in this case grabbing video frames and serving them over gRPC), and deploy broker services for obtaining the results of that action.
 
 ## Set up mock udev video devices
+1. Acquire an Ubuntu 20.04 LTS, 18.04 LTS or 16.04 LTS environment to run the
```

Comment:
Is ubuntu required for K3s and/or the v4l2loopback module?

Reply:
Quite possibly not, but have we tested it with anything else? For example the deb packages might work with Debian, but not much else.

Reply:
I'm okay with restricting it to ubuntu. I am thinking of adding a change where we clone and build the the v4l2loopback module from its repository, since they added a new feature of [dynamically adding and removing devices](https://github.com/umlaeute/v4l2loopback#dynamic-device-management). This might provide broader support too.